### PR TITLE
New package: TeamSpeakSystems.TeamSpeakClient.Beta version 5.0.0

### PR DESCRIPTION
**Pull request by rakleed**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135611)

### CODE REVIEW

File: manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TeamSpeakSystems.TeamSpeakClient.Beta
+PackageVersion: 5.0.0
+InstallerLocale: en-US
+InstallerType: wix
```
Comment (Trenly) [inline manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.installer.yaml:7]:
```suggestion
InstallerType: wix
FileExtensions: 
- kbn
- ts3_addon
- ts3_iconpack
- ts3_plugin
- ts3_soundpack
- ts3_style
- ts3_translation
```

File: manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: TeamSpeakSystems.TeamSpeakClient.Beta
+PackageVersion: 5.0.0
+PackageLocale: en-US
+Publisher: TeamSpeak
+PackageName: TeamSpeak
```
Comment (Trenly) [inline manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.locale.en-US.yaml:8]:
```suggestion
PackageName: TeamSpeak Beta
```
Necessary to not match control panel here to not break current installations of TeamSpeak 3

File: manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: TeamSpeakSystems.TeamSpeakClient.Beta
+PackageVersion: 5.0.0
+PackageLocale: en-US
+Publisher: TeamSpeak
```
Comment (Trenly) [inline manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.locale.en-US.yaml:7]:
```suggestion
Publisher: TeamSpeak
PublisherUrl: https://www.teamspeak.com/en/
PublisherSupportUrl: https://teamspeak.com/en/support/get-started/
PrivacyUrl: https://www.teamspeak.com/en/privacy-and-terms/
Author: TeamSpeak Systems GmbH
```

File: manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: TeamSpeakSystems.TeamSpeakClient.Beta
+PackageVersion: 5.0.0
+PackageLocale: en-US
+Publisher: TeamSpeak
+PackageName: TeamSpeak
+License: Proprietary
```
Comment (Trenly) [inline manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.locale.en-US.yaml:9]:
```suggestion
License: Proprietary, Freeware
LicenseUrl: https://www.teamspeak.com/en/features/licensing/
Copyright: © 2023. TeamSpeak Systems, Inc. All Rights Reserved.
CopyrightUrl: https://www.teamspeak.com/en/privacy-and-terms/
```

File: manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: TeamSpeakSystems.TeamSpeakClient.Beta
+PackageVersion: 5.0.0
+PackageLocale: en-US
+Publisher: TeamSpeak
+PackageName: TeamSpeak
+License: Proprietary
+ShortDescription: The all-new TeamSpeak Client is built on the foundations of our rock solid, lag-free voice technology and packed with next-generation communication & productivity tools, while maintaining our renowned security and privacy.
```
Comment (Trenly) [inline manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.locale.en-US.yaml:10]:
```suggestion
ShortDescription: The all-new TeamSpeak Client is built on the foundations of our rock solid, lag-free voice technology and packed with next-generation communication & productivity tools, while maintaining our renowned security and privacy.
Moniker: teamspeak-beta
Tags: 
- chat
- gaming
- ts
- ts3
- voice
- voice-chat
- voip
```

**Review by Trenly — CHANGES_REQUESTED** [review]
When adding beta packages, it's a good idea to bring forward the metadata from the stable release if it's still applicable

**Review by stephengillie — APPROVED** [review]

**Pull request merged**